src/Photos/Enums.cs

```diff
@@ -356,7 +358,9 @@ public enum PHProjectSectionType : long {
 	[Native]
 	[ErrorDomain ("PHLivePhotoEditingErrorDomain")]
 	public enum PHLivePhotoEditingError : long {
+		[Deprecated (PlatformName.MacOSX, 10, 15, message: "Use 'InternalError' instead.")]
```

Comment:
the suggested alternative is inside a different enum, which might be hard to use. Sadly not much we can do about it :(

However we should name that enum `Use 'PHPhotosError.InternalError' instead.`